packages/blockchain-wallet-v4-frontend/src/components/Flyout/InterestUploadDocuments/UploadAndVerify.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { IntlProvider } from 'react-intl'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import UploadAndVerify from './UploadAndVerify'
+
+export default {
+  argTypes: {},
+  args: {
+    documentLimits: {
+      categories: ['PROOF_OF_ADDRESS', 'PROOF_OF_INCOME', 'OTHER'],
+      maxAllowedFileSizeMBs: 5,
+      maxNumAllowedFiles: 8,
+      validTypes: ['jpg', 'png', 'heic']
+    },
+    nextStep: () => {},
+    previousStep: () => {},
+    submitData: (files) => {
+      // eslint-disable-next-line
+      console.log(files)
```

Comment:
It's not the end of the world if this stays in if it's useful during development since this story code isn't used in production it won't log to the actual console for users.

Reply:
yeah we do not need it